ntc_templates/templates/huawei_vrp_display_arp_all.textfsm

```diff
@@ -0,0 +1,11 @@
+Value Required IP_ADDRESS (\S+)
+Value Required MAC_ADDRESS (\S+)
```

Comment:
Looking to match the previous templates such that if you have a multivendor environment that this is normalized data already ahead of having to edit it. There are several templates that should be updated to match, but the predominate number of the templates are using the suggestion.

```suggestion
Value Required ADDRESS (\S+)
Value Required MAC (\S+)
```

Reply:
thanks for the suggestion